packages/cli/src/dioxus_crate.rs

```diff
@@ -25,7 +25,7 @@ pub(crate) static PROFILE_ANDROID: &str = "android-dev";
 pub(crate) static PROFILE_SERVER: &str = "server-dev";
 
 impl DioxusCrate {
```

Comment:
I'd like to merge this PR but I don't want to change the DioxusCrate::new() signature - I don't think the override stuff is worthwhile to include in this PR.

Can you strip this PR down to *just* fixing the always-on-top setting?